metricshub-hardware/src/main/java/org/sentrysoftware/metricshub/hardware/strategy/HardwarePostDiscoveryStrategy.java

```diff
@@ -40,6 +42,13 @@
 @EqualsAndHashCode(callSuper = true)
 public class HardwarePostDiscoveryStrategy extends AbstractStrategy {
 
+	// Those types of monitors belong to the enum {@link KnownMonitorType}.
+	// However, they are not included in the connector tag filtering.
+	private static final Set<String> EXCLUDED_MONITOR_TYPES = Set.of(

```

Comment:
Please remove this constant. And accept monitors of type `connector` that define a `hardware` tag and endpoint host monitors that are defined in a telemetry manager containing `connector` monitors having the `hardware` tag too.